### PR TITLE
Automate S3 bucket with lifecycle rules

### PR DESCRIPTION
**Pull request by bnshr**
This PR includes the code that are run to create the infrastructure to run collector.

* Updated code from my own repo
* Created user to access S3 bucket
* Create S3 bucket with secure access
* Add life cycle policy for deletion

### CODE REVIEW

File: stacksets/web.yaml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+      GroupDescription: SG to allow connection to the EC2
+      SecurityGroupIngress:
+        - IpProtocol: tcp
+          FromPort: '22'
+          ToPort: '22'
+          CidrIp: '0.0.0.0/0'
+        - IpProtocol: tcp
+          FromPort: '3000'
+          ToPort: '3000'
+          CidrIp: '0.0.0.0/0'
```
Comment (sebrandon1) [inline stacksets/web.yaml:45]: Above, you have the port `3000` section being deleted.  Is that the same here?

**Review by sebrandon1 — COMMENTED** [review]
One comment.  Also add newlines after the last line in each file so it doesn't flag them.

File: stacksets/web.yaml (hunk 1 above)
Comment (bnshr) [reply stacksets/web.yaml:45]: My local file does not have 3000 entry. This is weird.

Comment (bnshr) [reply stacksets/web.yaml]: Ah, thanks for the review. I removed the folder stacksets.

**Review by sebrandon1 — APPROVED** [review]

**Pull request merged**